content/appleboy/drone-ssh/index.md

```diff
@@ -40,39 +40,42 @@ pipeline:
       - echo world
 ```
 
-Example configuration for login with user private key:
+Example configuration for command timeout (unit: second), default value is 60 seconds:
 
 ```diff
 pipeline:
   ssh:
     image: appleboy/drone-ssh
     host: foo.com
     username: root
--   password: 1234
-+   key: ${DEPLOY_KEY}
+    password: 1234
     port: 22
++   command_timeout: 120
     script:
       - echo hello
       - echo world
 ```
 
-Example configuration for login with file path of user private key:
+Example configuration for execute commands on a remote server using ｀SSHProxyCommand｀:
 
 ```diff
 pipeline:
   ssh:
     image: appleboy/drone-ssh
     host: foo.com
     username: root
--   password: 1234
-+   key_path: ./deploy/key.pem
+    password: 1234
     port: 22
     script:
       - echo hello
       - echo world
++   proxy_host: 10.130.33.145
++   proxy_user: ubuntu
++   proxy_port: 22
++   proxy_password: 1234
 ```
 
-Example configuration for command timeout (unit: second), default value is 60 seconds:
+Example configuration for `master` branch:
```

Comment:
I would drop that, we are not documenting the conditions within the plugin index.

Reply:
Done.